lang/zh-cn.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label} 无效。",
 	"components.form-element.defaultFieldLabel": "字段",
 	"components.form-element.input.email.typeMismatch": "电子邮件无效",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {数字必须小于 {max}。} other {数字必须小于等于 {max}。}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {数字必须大于 {min}。} other {数字必须大于等于 {min}。}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {数目必须大于 {min} 且小于 {max}。} other {数目必须大于 {min} 且小于或等于 {max}。}}} other {{maxExclusive, select, true {数目必须大于或等于 {min} 且小于 {max}。} other {数目必须大于或等于 {min} 且小于或等于 {max}。}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {数目必须小于 {max}。} other {数目必须小于或等于 {max}。}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {数目必须大于 {min}。} other {数目必须大于或等于 {min}。}}",
```

Comment:
Before `Number must be greater than {min}.` After `The number must be greater than {min}.` same situation as above

Before `Number must be greater than or equal to {min}.` After is same